pages/client-libraries/go.mdx

```diff
@@ -1,130 +1,766 @@
+if err != nil {
+	panic(err)
+}
+// Process results 
+for _, rel := range result.Records {
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship))
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).GetProperties())
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).GetElementId())
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).Type)
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).StartElementId)
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).EndElementId)
+	fmt.Println(rel.AsMap()["r"].(neo4j.Relationship).Props["id"].(int64))
+}
+```
+You can access the `Relationship` properties in the same way as you access the [Node](#process-the-node-result) properties. `StartElementId` and `EndElementId` are the internal IDs of the start and end nodes of the relationship.
+
+#### Process the Path result
+
+You can receive path from the database, using the following construct:
+
+```go
+query = "MATCH p=(d:Developer)-[r:USES]->(t:Technology) RETURN p"
+result, err = neo4j.ExecuteQuery(ctx, driver, query, nil, neo4j.EagerResultTransformer, neo4j.ExecuteQueryWithDatabase(""))
+if err != nil {
+	panic(err)
+}
+
+for _, path := range result.Records {
+	fmt.Println(path.AsMap()["p"].(neo4j.Path))
+	fmt.Println(path.AsMap()["p"].(neo4j.Path).Nodes)
+	fmt.Println(path.AsMap()["p"].(neo4j.Path).Relationships)
+}
+```
+
+Path will contain [Nodes](#process-the-node-result) and [Relationships[#process-the-relationship-result], that can be accessed in the same way as in the previous examples, by casting them to the relevant type.
+
+
+#### Types mapping and casting
+
+ Here is the full table of the mapping between Memgraph Cypher types and the types used in the Go driver:
+
+|  Cypher Type | Driver Type        |
+|-------------:|:-------------------|
+|          Null| nil                |
+|       String | string             |
+|      Boolean | bool               |
+|      Integer | int64              |
+|        Float | float              |
+|         List | []any              |
+|          Map | map[string]any     |
+|         Node | neo4j.Node         |
+| Relationship | neo4j.Relationship |
+|         Path | neo4j.Path         |
+|     Duration | neo4j.Duration     |
+|         Date | neo4j.Date         |
+|    LocalTime | neo4j.LocalTime    |
+|LocalDateTime | neo4j.LocalDateTime|
+
+Keep in mind that Memgraph does not support timezones at the moment. 
 
-**4.** Add the **Bolt driver** with the command:
+### Transaction management
 
+Transaction is a unit of work that is executed on the database, it could be some basic read, write or complex set of steps in form of series of queries. There can be multiple ways to mange transaction, but usually, they are managed automatically by the driver or manually by the explicit code steps. Transaction management defines how to handle the transaction, when to commit, rollback, or terminate it.
```

Comment:
```suggestion
Transaction is a unit of work that is executed on the database, it could be some basic read, write or complex set of steps in form of series of queries. There can be multiple ways to mange transaction, but usually, they are managed automatically by the driver or manually by the explicit code steps. Transaction management defines how to handle the transaction, when to commit, rollback, or terminate it.

```